acceptance/lib/helper.rb

```diff
@@ -167,12 +167,7 @@ def upgrade_package(host, name)
   end
 
   def get_defaults_var(host, varname)
-    if master.is_pe?
```

Comment:
`is_pe?` is actually broken in the current Beaker, so this is a bit of a workaround.